charts/opentelemetry-collector/README.md

```diff
@@ -190,6 +190,91 @@ presets:
     extractAllPodAnnotations: true
 ```
 
+### Configuration for Annotation-Based Discovery
+
+The collector can be configured to automatically discover and collect telemetry from pods based on annotations. This feature provides a drop-in replacement for the `logsCollection` preset, allowing for selective collection of logs and metrics from specific pods rather than collecting from all pods in the cluster.
+
+> [!WARNING]
+> Annotation-based discovery and `logsCollection` are mutually exclusive. You should use either:
```

Comment:
`annotationDiscovery.logs` and `logsCollection` are mutually exclusive, not the whole `annotationDiscovery.*` :)